salt-master/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM centos:7.6.1810
+
+MAINTAINER moonshot-platform <moonshot-platform@scality.com>
+
+# Salt version to use
+ENV SALT_VERSION 2018.3-1
+
+# salt-master console logging log level
+ENV SALT_LOG_LEVEL warning
+
+# Install saltstack repository
+RUN yum install -y https://repo.saltstack.com/yum/redhat/salt-repo-${SALT_VERSION}.el7.noarch.rpm
+
+RUN yum install -y salt-master salt-api
```

Comment:
RUN creates additional layers each time it is invoked, better to squash them ?
also to minimize future diffs on Dockerfiles it is recommended to split commands on lines
```
RUN command1 \
    && command2 \
    && command3
```

Reply:
+1